codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperFuelWorker.res

```diff
@@ -3,10 +3,129 @@ open Belt
 
 exception EventRoutingFailed
 
+type event = {
+  name: string,
+  logId?: string,
+  mint?: bool,
+  eventMod: module(Types.Event),
+}
+
+type contract = {
+  name: string,
+  addresses: array<Address.t>,
+  events: array<event>,
+}
+
+let makeGetRecieptsSelectionOrThrow = (~contracts: array<contract>) => {
+  let logDataReceiptTypeSelection: array<Fuel.receiptType> = [LogData]
+  let mintReceiptTypeSelection: array<Fuel.receiptType> = [Mint]
+
+  // only transactions with status 1 (success)
+  let txStatusSelection = [1]
+
+  let mint: ref<[#None | #Wildcard | #NonWildcard]> = ref(#None)
+  let nonWildcardRbsByContract = Js.Dict.empty()
+  let wildcardRbs = []
+
+  let wildcardMintSelection = (
+    {
+      receiptType: mintReceiptTypeSelection,
+      txStatus: txStatusSelection,
+    }: HyperFuelClient.QueryTypes.receiptSelection
+  )
+  let maybeWildcardLogSelection = switch wildcardRbs {
+  | [] => None
+  | wildcardRbs =>
+    Some(
+      (
+        {
+          receiptType: logDataReceiptTypeSelection,
+          txStatus: txStatusSelection,
+          rb: wildcardRbs,
+        }: HyperFuelClient.QueryTypes.receiptSelection
+      ),
+    )
+  }
```

Comment:
Isn't this always the None case? I don't see where wildcardRbs gets any values added before this.

Reply:
True, caught this as well, while I writing tests